tests/store/test_sqlalchemy_store.py

```diff
@@ -352,40 +352,50 @@ def test_create_run_with_parent_id(self):
         expected = self._get_run_configs(
             name=run_name, experiment_id=experiment_id, parent_run_id=parent_run_id)
 
-        actual = self.store.create_run(**expected)
-
-        self.assertEqual(actual.info.experiment_id, experiment_id)
-        self.assertEqual(actual.info.user_id, expected["user_id"])
-        self.assertEqual(actual.info.name, run_name)
-        self.assertEqual(actual.info.source_type, expected["source_type"])
-        self.assertEqual(actual.info.source_name, expected["source_name"])
-        self.assertEqual(actual.info.source_version, expected["source_version"])
-        self.assertEqual(actual.info.entry_point_name, expected["entry_point_name"])
-        self.assertEqual(actual.info.start_time, expected["start_time"])
-
-        # Run creation should add two additional tags containing the run name and parent run id.
-        # Check for the existence of these two tags
-        self.assertEqual(
-            actual.data.tags, {MLFLOW_PARENT_RUN_ID: parent_run_id, MLFLOW_RUN_NAME: run_name})
-
-    def test_to_mlflow_entity(self):
-        # Create a run and obtain an MLflow Run entity associated with the new run
-        run = self._run_factory()
+        created_run = self.store.create_run(**expected)
+        fetched_run = self.store.get_run(created_run.info.run_uuid)
+
+        for actual in [created_run, fetched_run]:
```

Comment:
Check attributes of run returned by the write path & read path, just to be safe (this test failed before this PR as `experiment_id` was an int on the read path)

Reply:
LGTM. Might have added a test where experiment_id is passed as an int but it should not affect the to_mlflow_entity code path.